.circleci/config.yml

```diff
@@ -1,8 +1,25 @@
 version: 2
 jobs:
-  test:
+
+  test-build-golang-prev:
     docker:
       - image: circleci/golang:1.12
```

Comment:
Shouldn't you run the tests on `golang:1.13` too? 🤔

Reply:
I guess it couldn't hurt, but I haven't seen it be a problem yet, so we'll cross that bridge when we get to it :)